source/game/system/map/MapdataCannonPoint.hh

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "game/system/map/MapdataAccessorBase.hh"
```

Comment:
`include`s from the same directory (kart) should occur before those in a different directory (egg).
```suggestion
#include "game/system/map/MapdataAccessorBase.hh"

#include <game/kart/KartMove.hh>

#include <egg/math/Vector.hh>
```